packages/service-library/src/servicelib/fastapi/long_running_tasks/_context_manager.py

```diff
@@ -124,3 +126,11 @@ async def _wait_for_task_result() -> Any:
             timeout=task_timeout,
             exception=e,
         ) from e
+    except Exception as e:
+        error = TaskExceptionError(
+            task_id=task_id,
+            exception=e,
+            traceback=f"check remote side for logs, HINT: service replying to: '{client._base_url}' for '{task_id=}'",  # noqa: SLF001  # pylint:disable=protected-access
+        )
+        _logger.warning("%s", error)
```

Comment:
At risk of being pedantic (i have repeated these suggestions so many times) :-) ... 

1. IMO this should be logged by the service when the exception is handled, not in the library
2. A comprehensive way to log exceptions are using `create_troubleshotting_log_kwargs`.  Not only the log error is uniform but it has been curated for quite some time to give good diagnostic info. PLEASE use it.
3. There is no need to include the "exception in the exception". `raise error from e` does exactly that. Check in the python doc for `__cause__` attributes. Moreover, `create_troubleshotting_log_kwargs` takes care of that and shows you a short summary of the chain of exceptions that were raised. So PLEASE use it